From 3303272d7e64eabdf9bddf9c010292e10de53bf5 Mon Sep 17 00:00:00 2001
From: Francois Gouget <fgouget@codeweavers.com>
Date: Wed, 2 Nov 2016 16:38:05 +0100
Subject: [PATCH 07/55] vdagentd: Clarify the usage message

udsc is an internal library of sorts that's irrelevant to users.

Signed-off-by: Francois Gouget <fgouget@codeweavers.com>
---
 src/vdagentd/vdagentd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/vdagentd/vdagentd.c b/src/vdagentd/vdagentd.c
index 8dee414..74c372b 100644
--- a/src/vdagentd/vdagentd.c
+++ b/src/vdagentd/vdagentd.c
@@ -797,7 +797,7 @@ static void usage(FILE *fp)
             "  -h             print this text\n"
             "  -d             log debug messages (use twice for extra info)\n"
             "  -s <port>      set virtio serial port  [%s]\n"
-            "  -S <filename>  set udcs socket [%s]\n"
+            "  -S <filename>  set vdagent Unix domain socket [%s]\n"
             "  -u <dev>       set uinput device       [%s]\n"
             "  -f             treat uinput device as fake; no ioctls\n"
             "  -x             don't daemonize\n"
-- 
2.16.1

